src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -160,6 +191,12 @@ nginx_sites:
        proxy_next_upstream_tries: 1
        proxy_read_timeout: 900s
        limit_reqs:
+         - zone_name: "synced_old_bucket"
+           burst: 10
+           nodelay: True
```

Comment:
we can remove the burst and nodelay - they don't serve any purpose for these global limits (only useful for per IP / per user)

Reply:
updated.